Examples/Framework/include/ActsExamples/Framework/Sequencer.hpp

```diff
@@ -13,6 +13,7 @@
 #include "ActsExamples/Framework/IReader.hpp"
 #include "ActsExamples/Framework/IService.hpp"
 #include "ActsExamples/Framework/IWriter.hpp"
+#include "ActsExamples/Utilities/tbbWrap.hpp"
```

Comment:
I think `TbbWrap.hpp` would be most aligned with our naming guidelines.

Reply:
I wanted to keep the lower-case `tbb` in the namespace name `tbbWrap` to make it easier to grep both, but if the naming convention is more important, I can change it.

Or were you suggesting to change just `TbbWrap.hpp`, and leaving `namespace tbbWrap`?

Reply:
@paulgessinger, what do you think? Should I change `namespace TbbWrap` and/or `TbbWrap.hpp`, or leave it as it is?